File: Project/2024-spring-se102ta-project-se102a-f/Person.cpp
//
// Created by AAA on 4/18/2024.
//

#include "Person.h"
Person::Person(const string& name,int age):name(name),age(age){
}
string Person::getName(){
    return name;
}
int Person::getAge(){
    return age;
}
Person::Person() {}
void Person::setNameAndAge() {
    cout<<"Enter your name: ";
    cin>>name;
    cout<<"Enter your age: ";
    cin>>age;
}

File: assignments/2024-spring-se102a-assignment10-and-11-bsse23100/Person.py
class Person:
    def __init__(self, name, age):
        self._name = name
        self._age = age

    def getName(self):
        return self._name

    def getAge(self):
        return self._age

    def displayInfo(self):
        print("name=", self._name)
        print("age=", self._age)
